keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -903,6 +909,10 @@ Fundamental bugs may cause unexpected apiserver shutdowns or show up
 as 5xx error codes for operations involving `CSIStorageCapacity`
 objects.
 
+In Kubernetes <= 1.22, the `CSIDriver` object has to be deleted and
```

Comment:
This doesn't relate to the question `How can a rollout fail? Can it impact already running workloads?`

Reply:
True. It's more like background information. I can drop this chunk if mentioning it above is deemed sufficient.

Reply:
Removed.